sdk/python/test_data/test_data_config.yaml

```diff
@@ -144,6 +144,26 @@ pipelines:
     - module: components_with_optional_artifacts
       name: pipeline
       execute: false
+    - module: parallelfor_fan_in/fan_in_complex
+      name: math_pipeline
+      execute: false
+    - module: parallelfor_fan_in/pipeline_producer_consumer
+      name: math_pipeline
+      execute: false
+    - module: parallelfor_fan_in/simple_with_parameters
+      name: math_pipeline
+      execute: false
+    # TODO: uncomment to support artifact fan-in
+    # - module: parallelfor_fan_in/simple_with_artifacts
+    #   name: math_pipeline
+    #   execute: false
+    # TODO: uncomment to support artifact fan-in
+    # - module: parallelfor_fan_in/return_collected_artifacts_from_nested
+    #   name: math_pipeline
+    #   execute: false
```

Comment:
Remove until needed. The commented section is also incomplete--it doesn't reference `nested_with_artifacts` which presents in the PR, but references `return_collected_artifacts_from_nested` which doesn't present in the PR.

Reply:
Agreed. Updated. (And good catch. Demonstrates the reason for not leaving the comment in this PR.)